ord_schema/validations.py

```diff
@@ -3,26 +3,30 @@
 from ord_schema.proto import ord_schema_pb2 as schema
 
 import re
-from math import inf
-from dateutil import parser as dateparser
+import math
+import dateutil.parser
```

Comment:
Note that you can still do `from dateutil import parser`; I think `as dateparser` or `as dateutil_parser` would also be fine for clarity.